cross/fdupes/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME = fdupes
+
+NAME = fdupes
+PKG_DIST_SITE = https://github.com/adrianlopezroche/fdupes.git
+# From Makefile.inc/VERSION in the git checkout
```

Comment:
You probably should use official releases instead of using a git checkout for the same version. In this case, check  https://github.com/adrianlopezroche/fdupes/releases to check which releases are available, and use the associated download link, which is https://github.com/adrianlopezroche/fdupes/archive/fdupes-1.51.tar.gz